Exercise_D_E/index.html

```diff
@@ -1,16 +1,79 @@
-<!DOCTYPE html>
-<html class="no-js">
-  <head>
-    <meta charset="utf-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <title></title>
-    <meta name="description" content="" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <link rel="stylesheet" href="./assets/style/index.css" />
-  </head>
-  <body>
-    <!-- CODE HERE -->
-
-    <script src="" async defer></script>
-  </body>
-</html>
+<!DOCTYPE html>
+<html class="no-js">
+
+<head>
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <title></title>
+  <meta name="description" content="" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <link rel="stylesheet" href="./assets/style/index.css" />
+</head>
+
+<body>
+  <!-- CODE HERE -->
+  <div class="header">
+    <img src=".\assets\images\logo.png" alt="logo">
+  </div>
+
+<div class="body2">

```

Comment:
For this, you can move it and then change the height to 100vh and width to 100vw so that the light green occupies the whole page below